misc/net.minetest.minetest.appdata.xml

```diff
@@ -16,31 +16,31 @@
 		<p>
 			Minetest is an infinite-world block sandbox game and game engine.
 		</p><p xml:lang="de">
-			Minetest ist ein Sandkastenspiel und Spielengine mit unendlichen Welten
+			Minetest ist ein Sandkastenspiel und eine Spielengine mit unendlichen Welten.
 		</p><p>
 			Players can create and destroy various types of blocks in a
 			three-dimensional open world. This allows forming structures in
 			every possible creation, on multiplayer servers or in singleplayer.
 		</p><p xml:lang="de">
-			Spieler können in einer offenen 3D Welt viele verschiedene Arten von
+			Spieler können in einer offenen 3D-Welt viele verschiedene Arten von
 			Blöcken platzieren und abbauen. Dies erlaubt das Bauen von allen möglichen
 			Strukturen im Einzelspieler oder auf Mehrspielerservern.
```

Comment:
again I'm not 100%, ... im Einzelspieler- oder auf Mehrspieler-Servern.

Reply:
this might be a CH vs DE thing.

Reply:
alternatively: "im Einzel- oder Mehrspielermodus"